src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceColumn.java

```diff
@@ -939,9 +940,9 @@ private void manageCommands()
    public void newDoc(EditableFileType fileType,
                       ResultCallback<EditingTarget, ServerError> callback)
    {
-      ensureVisible(true);
```

Comment:
Don't we eventually need to call this so that e.g. the source pane opens for non-file tab types? (e.g. `View(mtcars)`)

Reply:
Yes, thank you. I had intended to change this back before submitting the PR.